assets/app/views/catalog/images.html

```diff
@@ -11,7 +11,8 @@
           <h1>Select a builder image</h1>
         </div>
       </div>
-      <div ng-if="!builders.length">There are no builder images to select from.  To add a builder to your project run <code>osc create -f &lt;image_repository_file&gt; -n {{projectName}}</code></div>
+      <div ng-if="!loadedProjectImageStreams && !loadedSharedImageStreams">Loading...</div>
```

Comment:
keep the empty messages nested inside an `ng-if="!builders.length"` block, one with `ng-if="loaded"`, one with `ng-if="!loaded"`